packages/frontend/editor-ui/src/components/RunDataJson.vue

```diff
@@ -184,7 +184,7 @@ const getListItemName = (path: string) => {
 				<template #renderNodeValue="{ node }">
 					<TextWithHighlights
 						v-if="isNaN(node.index)"
-						:content="getContent(node.content)"
+						:content="JSON.stringify(node.content)"
```

Comment:
Now we only do this for non-array items 🤔 Should we just modify getContent?

<img width="1305" alt="image" src="https://github.com/user-attachments/assets/914f4e6c-cf80-441a-9cf1-a8f773764dfb" />

Reply:
Missed this case. Yes, it should show escape chars on arrays as well.